### PR TITLE
Set Cali Offsets instantly

### PR DESCRIPTION
**Pull request by samuelballantyne**
This pull request addresses an issue where calibration values do not update immediately after the trigger is pulled during the calibration process.

### CODE REVIEW

File: SamcoEnhanced/SamcoEnhanced.ino [hunk 1]
```diff
@@ -1955,60 +2000,43 @@ void ExecCalMode()
...
-          break;
+            mouseTargetX = 32768 / 2;
+            mouseTargetY = 32768 / 2;
+            mouseMoving = true;
+            break;
         default:
-          break;
+            break;
     }
 }
```
Comment (SeongGino) [inline SamcoEnhanced/SamcoEnhanced.ino]: I know a lot of this is code I originally made, but if the only thing this function does is set init values, should we just put all these up in the ExecCalMode switch cases (after the cali values are set)? In retrospect, having this function where it is in the Calibration loop keeps tripping me up. :sweat_smile:

**Review by SeongGino — COMMENTED** [review]
Looks good, but just one little thing I had a thought about.

File: SamcoEnhanced/SamcoEnhanced.ino (hunk 1 above)
Comment (samuelballantyne) [reply SamcoEnhanced/SamcoEnhanced.ino]: Yeah, good point. The code has been updated to merge these switch cases into ExecCalMode.

**Review by SeongGino — APPROVED** [review]
Looks good to me! <3

**Pull request merged**